src/EdgeSwapping.cpp

```diff
@@ -94,11 +112,12 @@ void floatTetWild::edge_swapping(Mesh& mesh) {
 
         counter++;
     }
+    return 1;
 }
 
-bool floatTetWild::remove_an_edge_32(Mesh& mesh, int v1_id, int v2_id, const std::vector<int>& old_t_ids, std::vector<std::array<int, 2>>& new_edges){
+int floatTetWild::remove_an_edge_32(Mesh& mesh, int v1_id, int v2_id, const std::vector<int>& old_t_ids, std::vector<std::array<int, 2>>& new_edges){
     if(old_t_ids.size()!=3)
-        return false;
+        return 1;
```

Comment:
returning 1 here for false. in other places it's returning 0. is this intentional?